File: 05-11-2024_Minimum Number of Changes to Make Binary String Beautiful.py
def min_changes(s: str) -> int:
    changes = 0
    n = len(s)
    
    for i in range(0, n - 1, 2):  # Loop through even indices, comparing s[i] and s[i + 1]
        if s[i] != s[i + 1]:
            changes += 1
    
    return changes

File: 05-11-2024_Minimum Number of Changes to Make Binary String Beautiful.cpp
int minChanges(const std::string& s) {
    int changes = 0;
    int n = s.length();
    
    for (int i = 0; i < n - 1; i += 2) {
        if (s[i] != s[i + 1]) {
            changes++;
        }
    }
    
    return changes;
}